### PR TITLE
Support Kafka MirrorMaker 2.0 to replicate topics between Kafka clusters.

### PR DESCRIPTION
**Pull request by osiriswd**
### <Feature description>
- [x] Update the documentation to include this new feature.

- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

Kafka provides "MirrorMaker 2.0(MM2)" which leverages the Connect framework to replicate topics between Kafka clusters. This is realy helpful when handling large-scaled services/Multiple Data Centers/Edge Computing. 
When skywalking cluster deployed in a Data Center("DC1" for example) and the topics mirrored from other Data Centers will be renamed based on the aliases("DC2.skywalking-metrics etc."):
https://github.com/apache/kafka/tree/trunk/connect/mirror#remote-topics

Adding support to oap kafka module for the renamed topics, set the aliases via application.yml or environment variables. This feature need to set up an oap server for each mirrored Kafka cluster.

### CODE REVIEW

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java [hunk 1]
```diff
@@ -84,5 +84,29 @@
     private int kafkaHandlerThreadPoolSize;
 
     private int kafkaHandlerThreadPoolQueueSize;
-
+    
+    private String MM2SourceAlias = "";
+
+    private String MM2SourceSeparator = "";
```
Comment (wu-sheng) [inline oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java:90]: I can't find the place using these, have you submitted all codes?

File: docs/en/setup/backend/backend-fetcher.md [hunk 2]
```diff
@@ -122,3 +122,22 @@ kafka-fetcher:
       enable.auto.commit: true
       ...
 ```
+
+When useing Kafka MirrorMaker 2.0 to replicate topics between Kafka clusters, you can set the source Kafka Cluster alias(MM2SourceAlias) and separator(MM2SourceSeparator) according to your Kafka MirrorMaker config:
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-fetcher.md:126]:
```suggestion
When use Kafka MirrorMaker 2.0 to replicate topics between Kafka clusters, you can set the source Kafka Cluster alias(MM2SourceAlias) and separator(MM2SourceSeparator) according to your Kafka MirrorMaker config.
```

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java [hunk 3]
```diff
@@ -84,5 +84,29 @@
...
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfTracingSegments;
+    }
+
+    public String getTopicNameOfManagements() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfManagements;
+    }
+
+    public String getTopicNameOfMeters() {
+        return this.getMM2SourceAlias() + this.getMM2SourceSeparator() + this.topicNameOfMeters;
+    }
```
Comment (wu-sheng) [inline oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java:110]: Are you trying to hijack the get method? I recommend you don't do this. Put the logic in the provider, please.

**Comment by wu-sheng** [comment]
Changelog is required to update.

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java (hunk 3 above)
Comment (osiriswd) [reply oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java:110]: OK, working on it.

Comment (osiriswd) [reply oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java]: Moved to provider.

**Comment by wu-sheng** [comment]
Check CI logs, you are failing nearly all of them.

**Comment by wu-sheng** [comment]
Why do you change codes on the browser one by one, rather than local change? You are triggering our tests repeatedly, which cost many unnecessary resources. Please don't do this.

**Comment by osiriswd** [comment]
It's my fault...  Typing on my mobile phone, forget to switch to my fork. Won't change it anymore.

**Comment by wu-sheng** [comment]
Please finish the changes when you have your laptop back on hand. We are not hurried about the PR, especially now it is the weekend already.

We need to change codes, repackage, and run local tests to provide `application.yml`, logs, and screenshots of UI and influxDB data(with config). 
After that, we would ask at least one PMC member to verify this manually, as we don't have e2e tests to check automatically.

**Comment by wu-sheng** [comment]
Pending on @dmsolr review.

**Comment by dmsolr** [comment]
LGTM.
Thank you.

File: docs/en/setup/backend/backend-fetcher.md [hunk 4]
```diff
@@ -122,3 +122,21 @@ kafka-fetcher:
       enable.auto.commit: true
       ...
 ```
+
+When use Kafka MirrorMaker 2.0 to replicate topics between Kafka clusters, you can set the source Kafka Cluster alias(mm2SourceAlias) and separator(mm2SourceSeparator) according to your Kafka MirrorMaker config.
```
Comment (dmsolr) [inline docs/en/setup/backend/backend-fetcher.md:126]: typo.
mirror marker.
I recommend to add link refer to kafka mirror docs.

Comment (osiriswd) [reply docs/en/setup/backend/backend-fetcher.md]: Done.

**Review by wu-sheng — APPROVED** [review]

**Comment by dmsolr** [comment]
LGTM

**Review by dmsolr — APPROVED** [review]

**Pull request merged**